res/values-ar/strings.xml

```diff
@@ -155,7 +155,7 @@
   <string name="favorite">المفضلة</string>
   <string name="common_rename">إعادة التسمية</string>
   <string name="common_remove">حذف</string>
-  <string name="confirmation_remove_alert">هل تريد حقاً حذف %1$s ؟</string>
+  <string name="confirmation_remove_file_alert">هل تريد حقاً حذف %1$s ؟</string>
```

Comment:
String values are automatically update from "Transifex", so here in code we should only use the default (english) strings file.